napari/layers/surface/surface.py

```diff
@@ -220,6 +220,21 @@ def _get_extent(self):
 
         return [(min, max, 1) for min, max in zip(mins, maxs)]
 
+    def _get_state(self):
+        """Get dictionary of layer state.
+
+        Returns
+        -------
+        state : dict
+            Dictionary of layer state.
+        """
+        state = self._get_base_state()
+        state['colormap'] = self.colormap[0]
+        state['contrast_limits'] = self.contrast_limits
```

Comment:
same as above `contrast_limits_range`?  (in both cases... since it will not affect the visual representation, it's not critical, but it also probably doesn't hurt)

Reply:
see comment above about `init` of layer